ofrak_type/ofrak_type/memory_permissions.py

```diff
@@ -4,17 +4,22 @@
 class MemoryPermissions(Enum):
     """
     Representation of memory access permissions - all combinations of Read, Write, and eXecute
-    are represented, with the exception of Write + eXecute (not a sane combination).
+    are represented.
```

Comment:
@rbs-afflitto, ofrak_type changelog should be updated with this change, and the rc version bumped.

Reply:
See: <https://github.com/redballoonsecurity/ofrak/pull/583>